src/styles/watch.scss

```diff
@@ -0,0 +1,64 @@
+.watch {
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  height: 100vh;
+
+  &__space {
+    position: relative;
+    width: 500px;
+    height: 500px;
+    border-radius: 50%;
+    background-color: cornflowerblue;
+    border: 5px dotted black;
+  }
+
+  &__circle {
+    position: absolute;
+    top: 50%;
+    left: 50%;
+    transform: translate(-50%, -50%);
+    width: 15px;
+    height: 15px;
+    border-radius: 50%;
+    background-color: #000;
+    z-index: 2;
+  }
+
+  &__sec-hand {
+    position: absolute;
+    left: 50%;
+    transform: translate(-50%, -50%);
+    width: 5px;
+    height: 250px;
+    border-radius: 25%;
+    background-color: #000;
+    z-index: 1;
+    transform-origin: bottom center;
+    animation: count 60s linear infinite;
+
+  }
+
+  &__min-hand {
+    position: absolute;
+    top: 25%;
+    left: 50%;
+    transform: translateX(-50%, -50%);
```

Comment:
Nice clock =)
I noticed that clock hands do not stand clearly in the middle, so maybe try to transfer this value to the transition in the animation (also it should be `translateX(-50%)` because it's only an X axis). When I did this, minutes hands became centered, try to do the same with seconds hands
I had a similar problem and found the solution here:
https://stackoverflow.com/questions/36455370/css-animation-rotate-and-translate-doesnt-work-together
![image](https://user-images.githubusercontent.com/105058927/201139222-7777a755-eead-435b-becf-08e421e9ffe1.png)